### PR TITLE
Setup feature a

### PR DESCRIPTION
**Pull request by massaillon**
I have completed the Portfolio mobile version before the review for merge approval

### CODE REVIEW

**Review by zieeco — COMMENTED** [review]
# STATUS: INVALID!!!  :x: :x: :x: 

Hi @massaillon 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code, please make sure you implement the following changes before you resubmit for review. 

## Required Changes ♻️
_Check the comments below._

- [ ] As it is a requirement for this project,  please, make sure that [there are no linter error](https://github.com/microverseinc/linters-config)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by massaillon** [comment]
Hi @zieeco
There are no linters errors I triple checked from my local repository and ran many times the stylelint and the linter for checking HTML there were no errors nor warnings, moreover, I have passed all checks including the below :
1 - Linters / Lighthouse (pull_request) Successful in 1m 
2- Linters / Webhint (pull_request) Successful in 44s 
3- Linters / Stylelint (pull_request) Successful in 29s

File: index.html [hunk 1]
```diff
@@ -3,11 +3,55 @@
...
+
+                <div class="social-area">
+                    <div class="social-network">LET’S CONNECT</div>
+                    <div class="social-icon">
+                        <a href="#" class="twitter"><img src="../setup-and-mobile-version-skeleton/images/Twiter.svg" alt=""></a>
+                        <a href="#" class="linkedin"><img src="../setup-and-mobile-version-skeleton/images/Linkedin.svg" alt=""></a>
+                        <a href="#" class="M"><img src="../setup-and-mobile-version-skeleton/images/M.svg" alt=""></a>
+                        <a href="#" class="C"><img src="../setup-and-mobile-version-skeleton/images/C.svg" alt=""></a>
+                        <a href="#" class="2f"><img src="../setup-and-mobile-version-skeleton/images/2f.svg" alt=""></a>
+                    </div>
```
Comment (JuliCarracedo) [inline index.html]: - [x] Kindly move your social icons into <li> tags inside an <ul> tag. It is more semantically accurate and easier to handle.

**Review by JuliCarracedo — CHANGES_REQUESTED** [review]
Hi @massaillon , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ No linter errors
✔️ Professional README

## Required Changes ♻️
_Check the comments under the review._

- [ ] On your README, you are missing instructions on how to set up your project locally. Please add it.
- [ ] Currently, I can't see the styles or the local images being applied to the project. Kindly apply the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) you chose for this project.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (massaillon) [reply index.html]: @JuliCarracedo hi ok

Comment (massaillon) [reply index.html]: @JuliCarracedo what you mean, please

Comment (massaillon) [reply index.html]: @JuliCarracedo  hi Sir I have updated your review suggestions in the code please help to review it

**Comment by massaillon** [comment]
> Hi @massaillon ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
> ## Highlights
> 
> ✔️ No linter errors ✔️ Professional README
> ## Required Changes ♻️
> 
> _Check the comments under the review._
> 
>     * [ ]  On your README, you are missing instructions on how to set up your project locally. Please add it.
> 
>     * [ ]  Currently, I can't see the styles or the local images being applied to the project. Kindly apply the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) you chose for this project.
> 
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -3,11 +3,67 @@
...
-        <title>Hello Microverse</title>
-        <link rel="stylesheet" href="../Hello-Microverse/style.css">
+        <title>Setup and Mobile Version Skeleton</title>
+        <link rel="stylesheet" href="../setup-and-mobile-version-skeleton/style.css">
     </head>
-    <body>
-        <h1>Hello Microverse</h1>
+    <body >
+        <header id="header-container">
+            <div class="logo-container">My logo</div>
```
Comment (ShahierNashaat) [inline index.html:11]: - [x] [Optional] Kindly put the logo inside an `a` tag.

**Review by ShahierNashaat — APPROVED** [review]
Hi @massaillon , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (massaillon) [reply index.html:11]: @ShahierNashaat Thank you sir for the review
I will put the logo inside a tag as per your recommendation.

**Pull request merged**